models/labels/addresses/institution/identifier/cex/labels_cex_bitcoin.sql

```diff
@@ -17,3 +17,4 @@ SELECT blockchain
 , 'cex_' || blockchain AS model_name
 , 'identifier' AS label_type
 FROM {{ ref('cex_bitcoin_addresses') }}
+WHERE regexp_like(address, '^[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]+$')
```

Comment:
Hi @hildobby ,
do you have any idea how to get bitcoin addresses union with other `varbinary` typed addresses in `labels_cex`?

Currently I drop these 81 bitcoin addresses (with 1 being empty):
https://dune.com/queries/2812157

Reply:
I have no clue, I'm familiar with EVM stuff but much less on bitcoin data. Also, I this list was obtained from sources that aren't the most reliable, so feel free to start from scratch or even have it become empty, It's probably worth remaking entirely starting with addresses from CEXs proof of reserves for now